### PR TITLE
Fix .NET Core 3.X/.NET 5 WinForm Icon Issues

### PR DESCRIPTION
**Pull request by FoxCouncil**
This should resolve these two issues: #47 #46 

**TL;DR:** The issue stems from the order in which the new runtime in .NET Core initializes static fields.

**Details:** In `FormsIconHelper.FontFamilyFor()` the static field `FormsIconHelper.Fonts` is not accessed yet and thus does not load the actual packaged font data resources, causing the call to [Typeface.TryGetGlyphTypeface(GlyphTypeface)](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.typeface.trygetglyphtypeface?view=netcore-3.1) to fail as the font isn't actually loaded into memory/`PrivateFontCollection` yet.

**Commit Notes:** _Added a field accessor to trigger the field initialization. Get error checking for free!_

### CODE REVIEW

File: FontAwesome.Sharp/WinForms/FormsIconHelper.cs [hunk 1]
```diff
@@ -118,6 +118,7 @@ public static void AddIcons(this ImageList imageList, int size, Color color, par
 
         internal static FontFamily FontFamilyFor(IconChar iconChar)
         {
+            if (Fonts == null) throw new Exception("FontAwesome source font files not found!");
```
Comment (mkoertgen) [inline FontAwesome.Sharp/WinForms/FormsIconHelper.cs]: This is the fix? Ah ok, the exception probably makes WinForms re-evaluate/-load the fonts at a later time in the app lifecycle.

Ok, let's merge it in then. I would probably change it to a more specific exception, e.g. `InvalidOperationException`

Comment (FoxCouncil) [reply FontAwesome.Sharp/WinForms/FormsIconHelper.cs]: `if (Fonts ==` 

Just accessing the `Fonts` static field causes the initializer `InitializeFonts();` to be called, making sure that the font data is loaded into memory in the app lifecycle before trying to determining if a glyph is in any of the fonts.

Also changed the exception to `InvalidOperationException`.

I agree, let's merge it in!

**Comment by FoxCouncil** [comment]
@mkoertgen Just as a note, I can't merge this PR as I don't have write access to this repo. You'd have to click the button.
